.circleci/config.yml

```diff
@@ -69,7 +69,7 @@ jobs:
           name: Install algoliasearch v3
           command: |
             yarn remove -W @algolia/client-search
-            yarn add -W @types/algoliasearch@3.34.8 algoliasearch@3.35.1
```

Comment:
don't we need to test typescript on v3 with the right type?

Reply:
Good catch.

Reply:
No sure to understand here, what to you mean?

Reply:
If we're testing v3 types, then we need to install @types/algoliasearch, to have a correct setup I think

Reply:
Ho ok. I added the types to the setup to test with algoliasearch v3.

I also had to remove the examples before that, since they are relying on algoliasearch v4 we had both algoliasearch v3 and v4 in the lockfile, messing with the test.